micall/tests/test_sample.py

```diff
@@ -172,3 +177,64 @@ def test_exclude_extra_seeds(project_code, excluded, expected):
     all_excluded = exclude_extra_seeds(excluded, project_code)
 
     assert all_excluded == expected
+
+
+def test_context_manager(tmp_path):
+    file_path = os.path.join(tmp_path, 'testfile.csv')
+    file_info = (file_path, 'w')
+    with open_files(testfile=file_info) as opened_files:
+        file = opened_files['testfile']
+        assert isinstance(file, io.TextIOBase)
+
+    assert file.closed
+
+
+def test_context_manager_write(tmp_path):
+    file_path = os.path.join(tmp_path, 'testfile.csv')
```

Comment:
No need to change it, but `tmp_path` gives you a `Path` object. That means you could use `tmp_path / 'testfile.csv'` instead of `os.path.join(...)`. It also makes it easier to read the file content.